### PR TITLE
Move adminCenter-1.0 to OL

### PR DESCRIPTION
**Pull request by fritze2**

### CODE REVIEW

File: dev/com.ibm.ws.ui/resources/com/ibm/ws/ui/internal/resources/UIMessages.nlsprops [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+LOADED_PERSISTED_CATALOG=CWWKX1006I: The Admin Center catalog loaded.
+LOADED_PERSISTED_CATALOG.explanation=The Admin Center catalog loaded from persisted storage. 
+LOADED_PERSISTED_CATALOG.useraction=No action is required.
+
+TOOL_ALREADY_EXIST=CWWKX1007E: Tool {0} already exists in the Admin Center catalog.
+TOOL_ALREADY_EXIST.explanation=The catalog already contains a tool with the specified tool ID.
+TOOL_ALREADY_EXIST.useraction=A tool with the same name and version already exists in the catalog. Specify an alternate tool name (or version) and try again.
+TOOL_ALREADY_EXIST.developeraction=A tool with that ID already exists in the catalog. If you are trying to add a new tool, change either the tool name or version. If you are trying to update the tool, modify the existing tool entry with a PUT request.
+
+TOOL_OBJECT_NOT_VALID=CWWKX1008E: The tool will not be added to the Admin Center catalog. The tool object does not pass validation rule: {0}
```
[the remark this answers is not in the document]
Comment (fritze2) [reply dev/com.ibm.ws.ui/resources/com/ibm/ws/ui/internal/resources/UIMessages.nlsprops:54]: Should this message be "is not added" instead?

**Comment by fritze2** [comment]
#build

File: dev/com.ibm.ws.ui/resources/com/ibm/ws/ui/internal/resources/UIMessages.nlsprops (hunk 1 above)
[the remark this answers is not in the document]
Comment (Rwalls1) [reply dev/com.ibm.ws.ui/resources/com/ibm/ws/ui/internal/resources/UIMessages.nlsprops:54]: Yes, sorry, it should be "is not added"

**Comment by fritze2** [comment]
#build

**Comment by k8vance88** [comment]
#build

**Comment by k8vance88** [comment]
#build

**Comment by k8vance88** [comment]
#build

File: dev/ant_build.js/build.gradle [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+    libs6
+    libs7
+    libs8
+    libs9
+    libs10
+}
+
+dependencies {
+    libs "com.fasterxml.jackson.core:jackson-annotations:2.2.3"
+    libs "com.fasterxml.jackson.core:jackson-core:2.2.3"
```
Comment (tevans78) [inline dev/ant_build.js/build.gradle:30]: I don't like having all these version numbers listed here. Many of these are old versions and will be lacking various fixes. jackson for example was recently updated to pick up an important fix; https://github.com/OpenLiberty/open-liberty/pull/14168

Comment (tevans78) [reply dev/ant_build.js/build.gradle:30]: Also, this project isn't really ant based is it? Shouldn't it be given a different name?

**Comment by k8vance88** [comment]
#build

**Comment by k8vance88** [comment]
#build

File: dev/ant_build.js/build.gradle (hunk 2 above)
Comment (tevans78) [reply dev/ant_build.js/build.gradle:30]: @k8vance88 @fritze2 any thoughts on this?

**Comment by k8vance88** [comment]
Closing this because I have it covered under a new one: https://github.com/OpenLiberty/open-liberty/pull/15078

**Pull request closed without merging**